web/massastation/src/mirage/server.js

```diff
@@ -159,7 +159,7 @@ function mockServer(environment = ENV.DEV) {
         return new Response(200, {});
       });
 
-      this.post('plugin-manager', (schema, request) => {
+      this.post('plugin-manager', { timing: 2000 }, (schema, request) => {
```

Comment:
don't push this, I thinks

Reply:
why ?

Reply:
I thought we didn't change mirage. But okay

Reply:
removed